course/include/Vector3.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+#include <iostream>
+#include "math.h"
+
+namespace ekumen
+{
+
+class Vector3
+{
+public:
+    explicit Vector3();
+    explicit Vector3(const double x, const double y, const double z);    
```

Comment:
Apply the rule-of-five and create move operations: construct and assign. (this is why I wanted an allocated storage).